config/Reader.debug.xcconfig

```diff
@@ -1,6 +1,8 @@
-#include "Common.alpha.xcconfig"
+#include "Common.debug.xcconfig"
```

Comment:
@kean I assumed the debug and release `xcconfig` including the alpha common settings was a copy pasta. But let me know if it was intentional and we'll find a way to adjust. Thanks!